tests/test_lab_serve.py

```diff
@@ -335,3 +335,19 @@ def test_warn_for_unsupported_backend_param(param, expected_call_count):
             mock_warning.assert_called_with(
                 f"Option '--{param.replace('_','-')}' not supported by the backend."
             )
+
+
+def test_host_port(cli_runner: CliRunner):
+    args = common.vllm_setup_test(
+        cli_runner,
+        [
+            "--config=DEFAULT",
+            "model",
+            "serve",
+            "--model-path=foo",
+            "--host-port",
+            "192.168.1.1:8000",
```

Comment:
This logic looks correct, though I would advise you to use a more obscure address & port here just so we know it's truly reading this. Because `8000` is otherwise fairly common and could be getting populated from the defaults. 

```suggestion
            "2.4.6.8:1738",
```

Reply:
@osilkin98 thanks, updated